ingestion/src/metadata/ingestion/source/dashboard/quicksight/metadata.py

```diff
@@ -169,6 +171,26 @@ def yield_dashboard(
         yield Either(right=dashboard_request)
         self.register_record(dashboard_request=dashboard_request)
 
+    def _parse_sql_query(self, sql_query: str) -> Optional[tuple]:
+        try:
+            parsed = sqlparse.parse(sql_query)[0]
+            tokens = parsed.tokens
+
+            from_read=False
+            for token in tokens:
+                if token.ttype is Keyword and token.value.upper() == 'FROM':
+                    from_read = True
+                    continue
+                if from_read and isinstance(token, sqlparse.sql.Identifier):
+                    # jump to identifier only after from clause
+                    identifier = token.get_real_name()
+                    table_name = token.get_name()
+                    schema_name = token.get_parent_name()
+                    return table_name, schema_name
+        except Exception as err:
+            logger.info(f"Error to parse query:{sql_query}")
+            return None
+
```

Comment:
this is now the way how we extract lineage for dashboard queries, please refer to how we do it for other dashboard connectors https://github.com/open-metadata/OpenMetadata/blob/5e80ad9fc379fc5f6860e358029e3b8673d000f5/ingestion/src/metadata/ingestion/source/dashboard/metabase/metadata.py#L322